src/acquisition/paho/paho_download.py

```diff
@@ -78,10 +80,13 @@ def get_paho_data(offset=0, dir='downloads'):
     # Extract session ID
     href = downloadbutton.get_attribute("href")
     startidx = href.index("sessions/") + len("sessions/")
-    endidx = href.index("/",startidx)
+    endidx = href.index("/", startidx)
     sessionid = href[startidx:endidx]
 
-    dataurl = "http://phip.paho.org/vizql/w/Casosdedengue_tben/v/ByLastAvailableEpiWeek/viewData/sessions/%s/views/18076444178507886853_9530488980060483892?maxrows=200&viz=%%7B%%22worksheet%%22:%%22W%%20By%%20Last%%20Available%%20EpiWeek%%22,%%22dashboard%%22:%%22By%%20Last%%20Available%%20Epi%%20Week%%22%%7D"%sessionid
+    dataurl = (
+        "https://phip.paho.org/vizql/w/Casosdedengue_tben/v/ByLastAvailableEpiWeek/viewData/sessions/%s/views/18076444178507886853_9530488980060483892?maxrows=200&viz=%%7B%%22worksheet%%22:%%22W%%20By%%20Last%%20Available%%20EpiWeek%%22,%%22dashboard%%22:%%22By%%20Last%%20Available%%20Epi%%20Week%%22%%7D"
```

Comment:
suggestion: Consider replacing with an f-string.

Reply:
if you think it's relevant enough. I saw some other string formats converted to f-strings.